client/src/components/BonusItems.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+const BonusItems = (props) => {
+  const albums = props.bonus.bonus_info;
+
+  const bonusAlbums = albums.map((album, index) => {
+    return (
+      <div className='bonus-album' key={index}>
```

Comment:
I don't remember your schema well enough, but if every `album.title` is unique, I think it would be better to use that as the unique key as React recommends using the [item index as a key as a last resort](https://reactjs.org/docs/lists-and-keys.html#keys).

Reply:
Changed.